pkg/volume/empty_dir/empty_dir.go

```diff
@@ -168,26 +206,66 @@ func (ed *emptyDir) setupTmpfs(dir string) error {
 	// By default a tmpfs mount will receive a different SELinux context
 	// from that of the Kubelet root directory which is not readable from
 	// the SELinux context of a docker container.
-	//
-	// getTmpfsMountOptions gets the mount option to set the context of
-	// the tmpfs mount so that it can be read from the SELinux context of
-	// the container.
-	opts := ed.getTmpfsMountOptions()
-	glog.V(3).Infof("pod %v: mounting tmpfs for volume %v with opts %v", ed.podUID, ed.volName, opts)
+	var opts []string
+	if selinuxContext != "" {
```

Comment:
I agree.  I'm actually starting to gather reqs on solving this for other volumes, and I think we can have a follow-up to push the SELinux code up to a higher level so that the volume plugins won't have to do this work individually.  Are you cool w/ that @thockin ?